management/virtualmachine/client.go

```diff
@@ -118,6 +119,25 @@ func (vm VirtualMachineClient) GetRole(cloudServiceName, deploymentName, roleNam
 	return role, nil
 }
 
+// AddRole adds a Virtual Machine to a deployment of Virtual Machines, where role name = VM name
+// See https://msdn.microsoft.com/en-us/library/azure/jj157186.aspx
+func (vm VirtualMachineClient) AddRole(cloudServiceName string, deploymentName string, role Role) (management.OperationID, error) {
+	if cloudServiceName == "" {
```

Comment:
That's an ongoing issue with go tool / git combo in Travis started today. https://github.com/golang/go/issues/13206

We can ignore it if it compiles on your machine.